examples/components/Radar.vue

```diff
@@ -26,34 +37,83 @@ export default {
       default: {
         direction: 0
       }
+    },
+    handler: {
+      type: Object,
+      default: {
+        draggable: true
+      }
     }
   },
   watch: {
-    'config.angle': function (newVal, oldVal) {
-      this.handleAngleChange(true);
+    'config.angle': function() {
+      this.handleAngleChange()
+    },
+    'config.direction': function() {
+      this.handleAngleChange()
+    },
+    'config.x': function() {
+      this.resetHandler()
     },
-    'config.direction': function (newVal, oldVal) {
-      this.handleDirectionChange(false);
+    'config.y': function() {
+      this.resetHandler()
```

Comment:
maybe `resetHandler()` can be done directly on `onDragEnd()`?

Reply:
yes right, now x and y no need be watched